guides/reference/windows.md

```diff
@@ -1 +1,83 @@
-<!-- Port over info from existing site -->
\ No newline at end of file
+Windows Vista and newer windows versions are fully supported.
+
+To get started ensure the following dependencies are installed:
+
+* Node.js - [https://nodejs.org/en/](https://nodejs.org/en/)
+* Git - [https://git-scm.com/](https://git-scm.com/)
+* Phantom.js - [http://phantomjs.org/](http://phantomjs.org/)
+
+### Performance
+
+Although supported, Windows performance, at least by default, isn't as good as
+on Linux or MacOS. On a positive note, this story continues to improve. Both
+Microsoft, and the Ember CLI team continue to work to improve these developer
+ergonomics.
+
+#### What causes the build slowdown?
+
+The two primary reasons are:
+
+* Lack of enabled-by-default symlinks
+* Generally slower FS operations on NTFS
+
+#### For the best possible Windows experience
+
+* Windows 10, insiders release with development mode enabled. Symlinks are enabled
```

Comment:
I'm not familiar with Windows much, but I was wondering what an `insiders release` is. Do you think it would be relevant to explain this in more detail here or link to a resource where users can read more about it?

Reply:
I think this part is out of date. I didn't edit this page. Can someone with knowledge confirm?

Reply:
This part is *long* out of date; you just need use a current version of Windows.

A few other quick notes:

- For performance in general, we should recommend people run the command prompt or PowerShell instance as administrator. (If you can't run as admin, you're going to have a very bad on time on Windows, symlinks dance or no.)
- `ember windows` is basically [borked](https://github.com/felixrieseberg/ember-cli-windows/issues/39) for many users at this point; [this workaround](https://github.com/felixrieseberg/ember-cli-windows/issues/40#issuecomment-363902061) may or may not help. (We need someone to take over and update ember-cli-windows.)

Reply:
Do we want to mention the workaround or just leave it alone for now?

Reply:
Removing reference to insider's build.

Reply:
Removed reference to insider's build, changed instructions to reflect @chriskrycho 's comments.